### PR TITLE
Krnlmon fixes for LAG and ECMP defects

### PR DESCRIPTION
**Pull request by n-sandeep**
Modify p4 program mapping file with correct table, key and action names

### CODE REVIEW

File: switchapi/es2k/switch_pd_lag.c
Comment (ffoulkes) [inline switchapi/es2k/switch_pd_lag.c:161]: Shouldn't this identifier be the same as the one used in the preceding function call?

**Review by ffoulkes — APPROVED** [review]
I think you've specified the wrong key at line 161 in es2k/switch_pd_lag.c.

I don't want to block the PR, so I'm going to approve it on the assumption that you will correct the error before merging it.

If it _isn't_ an error -- if you really did intend to use a different identifier -- please add a comment saying that this is intentional (and why). Otherwise, some future maintainer is going to think it's a bug and "fix" it. (I know I would.) 🙂

**Review by aashishkuma — APPROVED** [review]
LGTM

**Review by vsureshkumarp — APPROVED** [review]
LGTM...:)

**Pull request merged**
